x/stakeibc/keeper/icqcallbacks_withdrawal_reward_balance.go

```diff
@@ -25,10 +25,6 @@ func WithdrawalRewardBalanceCallback(k Keeper, ctx sdk.Context, args []byte, que
 		"Starting withdrawal reward balance callback, QueryId: %vs, QueryType: %s, Connection: %s", query.Id, query.QueryType, query.ConnectionId))
 
 	chainId := query.ChainId
-	hostZone, err := k.GetActiveHostZone(ctx, chainId)
```

Comment:
Why remove this?

Reply:
the hostZone struct is not used anymore in this function